openlibrary/templates/work_search.html

```diff
@@ -80,6 +80,7 @@
 }
 window.q.push(function(){
     \$(".header_bull").hide();
+    \$(".header_less").hide();
```

Comment:
Can I ask that in light of #4474 that we also tackle the inline script while here.

It's also a bit of an anti-pattern to hide something that is going to be shown in JS. It would be better to /create/ this element in JavaScript and insert it, rather than do this which results in a flash of unstyled content. That said that's out of scope for this.

However,  the `.ui-helper-hidden` class exists for exactly this purpose. Please add that class to the relevant HTML to get the same effect and remove it in JS where needed.

Reply:
Done.